data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -102,6 +104,8 @@
 
         no_leader=yes
         hidden=yes
+        
+        {VARIABLE undead no}
```

Comment:
Only once, in prestart (usually) or  start.

Reply:
Sorry didn't actually notice these were two variables.

Reply:
Everything should be in place now!